src/br/ufla/gac106/s2022_1/seuJogo/Jogo.java

```diff
@@ -83,20 +83,20 @@ private void imprimirBoasVindas() {
         );
         System.out.println("Digite 'ajuda' se voce precisar de ajuda.");
         System.out.println();
+        
+        System.out.println("Você está " + ambienteAtual.getDescricao());
 
-        System.out.println("Voce esta " + ambienteAtual.getDescricao());
-
-        System.out.print("Saidas: ");
-        if (ambienteAtual.saidaNorte != null) {
+        System.out.print("Saídas: ");
+        if (ambienteAtual.getSaida(Direcao.NORTE) != null) {
             System.out.print("norte ");
         }
-        if (ambienteAtual.saidaLeste != null) {
+        if (ambienteAtual.getSaida(Direcao.LESTE) != null) {
             System.out.print("leste ");
         }
-        if (ambienteAtual.saidaSul != null) {
+        if (ambienteAtual.getSaida(Direcao.SUL) != null) {
             System.out.print("sul ");
         }
-        if (ambienteAtual.saidaOeste != null) {
+        if (ambienteAtual.getSaida(Direcao.OESTE) != null) {
             System.out.print("oeste ");
         }
```

Comment:
Esse trecho e outro, mais abaixo, originalmente tinham o mesmo código e, agora com as alterações, cada um deles foi tratado de uma forma. Seria melhor se os dois trechos tivessem o mesmo tratamento. Creio que o segundo tratamento que usa `for (Direcao direcao : Direcao.values())` seja melhor. Portanto, seria interessante fazê-lo aqui também.